packages/expo-asset/src/AssetSourceResolver.ts

```diff
@@ -0,0 +1,3 @@
+import AssetSourceResolver from 'react-native/Libraries/Image/AssetSourceResolver';
+export default AssetSourceResolver;
+export * from 'react-native/Libraries/Image/AssetSourceResolver';
```

Comment:
Maybe it's because RN still uses CommonJS, but does this one line not take care of exporting `default` as well?

Reply:
Just tried this, it doesn't work :/